docs/app-starter.stories.mdx

```diff
@@ -53,17 +53,19 @@ The order of these CSS stylesheets is **very important**:
 
 No matter how you choose to import the following, this order must be upheld.
 
+**We recommend you use your `tailwind.css` file to ensure this order (JS files will order them differently)**
+
 ### Reset
 
 Kaizen's CSS reset aims to add any global default styles such as typography or base styles which enable Tailwind utilities (you can think of this as Kaizen's own Preflight).
```

Comment:
I suggest linking this mention of Preflight to the Tailwind docs on the subject, since many readers will be unfamiliar with this: https://tailwindcss.com/docs/preflight